server/models/Profile.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  address: {
+    type: String,
+    required: true,
+  },
+  city: String,
+  phoneNumber: Number,
+  profilePic: String,
+  aboutMe: String,
+  isSitter: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  availability: [
+    {
+      start: Date,
+      end: Date,
+    },
+  ],
```

Comment:
This works initially however, might require some changes when integrating into a calendar in the front-end. 👍🏻